packages/devtools_testing/pubspec.yaml

```diff
@@ -2,16 +2,16 @@ name: devtools_testing
 description: Package of shared testing code for Dart DevTools.
 # Note: this version should only be updated by running tools/update_version.sh
 # that updates all versions of packages from packages/devtools.
-version: 0.9.8-dev.1
+version: 0.9.8-dev.2
```

Comment:
nit: any reason why the version is updated as part of this CL?

Reply:
No, since I had to update a few packages I ran the update script which updated all packages.